menu/cakemenu.html

```diff
@@ -0,0 +1,270 @@
+<!DOCTYPE HTML>
+<html>
+	<head>
+		<title>mandy b. bakes</title>
+		<meta property="og:title" content="Mandy B Bakes" />
+		<!-- <meta property="og:image" content="images/bg.jpg" /> -->
+		<meta property="og:description" content="Amanda B's Online Bakery" />
+		<meta property="og:url" content="https://yu-huanwu.github.io/mandybbakes/" />
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+		<link rel="stylesheet" href="/assets/css/main.css" />
```

Comment:
notice how this href is different from index.html's. That's because all the menu are one folder deep, so to access "assets/css/main.css", we need to add "/" in front to show the absolute path.